.github/workflows/main.yml

```diff
@@ -203,13 +185,23 @@ jobs:
             const taquito = readInfo("./taqueria-plugin-taquito/package.json")
             const sdk = readInfo("./taqueria-sdk/package.json")
 
+            const git_short_sha = "${{ github.sha }}".substring(1, 8)
+            const git_tag = "${{ github.ref }}".split('tags/')[1];
+
             let body = `
+
+
+              | Git Commit | Artifacts |
+              | ---------- | --------- |
+              | ${git_short_sha} | [Linux](${{ needs.build-binaries.outputs.public-url-ubuntu }}) <br /> [MacOS](${{ needs.build-binaries.outputs.public-url-macos }}) <br /> [Windows](${{ needs.build-binaries.outputs.public-url-windows }}) |
+
+
               | npm package installation instructions |
-              | --------- |
-              | npm install ${{ env.SCOPE }}/${ligo.name}@${ligo.version} --registry=${{ env.GH_REGISTRY }}|
-              | npm install ${{ env.SCOPE }}/${smartpy.name}@${smartpy.version} --registry=${{ env.GH_REGISTRY }}|
-              | npm install ${{ env.SCOPE }}/${taquito.name}@${taquito.version} --registry=${{ env.GH_REGISTRY }}|
-              | npm install ${{ env.SCOPE }}/${sdk.name}@${sdk.version} --registry=${{ env.GH_REGISTRY }}|
+              | ------------------------------------- |
+              | npm install @taqueria/${ligo.name}@${ligo.version}-${{ github.event.number }}-${git_short_sha}|
+              | npm install @taqueria/${smartpy.name}@${smartpy.version}-${{ github.event.number }}-${git_short_sha}|
+              | npm install @taqueria/${taquito.name}@${taquito.version}-${{ github.event.number }}-${git_short_sha}|
+              | npm install @taqueria/${sdk.name}@${sdk.version}-${{ github.event.number }}-${git_short_sha}|
```

Comment:
Why are you hardcoding this? referencing a common variable is usually preferred

Reply:
Thanks for pointing that out, I reverted this change